.github/actions/e2e-quantic-setup/action.yml

```diff
@@ -13,9 +13,9 @@ runs:
     - run: npx nx run quantic:build
       shell: bash
     - run: |
-        echo "${{ inputs.jwtkey }}" > ${{ env.SFDX_AUTH_JWT_KEY_FILE }}
+        echo "${{ inputs.jwtkey }}" > server.key
         npx --no-install ts-node scripts/build/deploy-community.ts --ci
-        rm ${{ env.SFDX_AUTH_JWT_KEY_FILE }}
+        rm server.key
```

Comment:
I set this as an environment variable in Quantic Staging / Production, but for e2e we can just use a constant. It's not sensitive so it doesn't have to be a repo secret

Reply:
TODO: remove the `SFDX_AUTH_JWT_KEY_FILE` and `SFDX_AUTH_JWT_USERNAME` repo secrets